server/server.go

```diff
@@ -620,6 +622,7 @@ func (s *Server) initRouters() {
 		}
 
 		router.Handle("/health", s.instrumentHandler(s.unversionedGetHealth, PromHandlerHealth)).Methods(http.MethodGet)
+		router.Handle("/health/{path:.+}", s.instrumentHandler(s.unversionedGetHealthWithPolicy, PromHandlerHealth)).Methods(http.MethodGet)
```

Comment:
Wondering if we want to expose every rule under the health package (and all subpackages, recursively) or just a small number of well-defined endpoints... I realize there's more flexibility granted by this approach but I think it would also be simpler for uneducated users if we could just point them at (`/system/health/ready`, `/system/health/live`, etc.) Similarly, someone looking at an OPA deployment might be confused by the more elegant solution.

Reply:
What if the documentation/comments indicate the ready/live rules as the convention, but we don't actually lock down the route?  I can see the need for unique healthcheck behaviors based on which orchestration system is in use.  Flexibility in the route doesn't seem mutually exclusive with making it simple to understand/use.  Thoughts?

Reply:
> I can see the need for unique healthcheck behaviors based on which orchestration system is in use.

That seems reasonable. Let's solve this in the documentation with examples like you suggested.